dlink-doc/sql/dinky.sql

```diff
@@ -110,6 +110,8 @@ CREATE TABLE `dlink_cluster` (
                                  `cluster_configuration_id` int DEFAULT NULL COMMENT 'cluster configuration id',
                                  `task_id` int DEFAULT NULL COMMENT 'task ID',
                                  `enabled` tinyint(1) NOT NULL DEFAULT '1' COMMENT 'is enable',
+                                 `application_id` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci DEFAULT NULL COMMENT 'Job Manager Host',
+                                 `resource_manager_addr` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci DEFAULT NULL COMMENT 'Job Manager Host',
```

Comment:
The comment of application_id is wrong.
The comment of resource_manager_addr is wrong.